src/buildstream/_frontend/cli.py

```diff
@@ -538,6 +540,7 @@ def build(
 @click.option(
     "--except", "except_", multiple=True, type=click.Path(readable=False), help="Except certain dependencies"
 )
+@click.option("--json", "as_json", is_flag=True, help="Output the information as machine readable JSON")
```

Comment:
It would be nice to have a similar option that outputs YAML, as that's what buildstream uses.